lib/nimble_totp.ex

```diff
@@ -167,14 +193,25 @@ defmodule NimbleTOTP do
   @doc """
   Checks if the given `otp` code matches the secret.
 
-  It accepts the same options as `verification_code/2`.
+  ## Options
```

Comment:
```suggestion
  ## Options

```

 Nit for consistency. :)